priv/js/graph.js

```diff
@@ -57,11 +76,52 @@ function initializeGraph() {
       }
     }),
   })
-  window.r = renderer
-
   renderer.run()
 }
 
+window.c = createPopup
+
+function createPopup(node) {
+  const {
+    id,
+    data: {
+      description,
+    },
+  } = node
+
+  const f = document.createElementNS('http://www.w3.org/2000/svg', 'foreignObject')
```

Comment:
You don't even want to know to what depths of frontend hell I had to travel to get this working...